account_invoice_auto_send_by_email/tests/test_account_invoice_auto_send_by_email.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+import mock
+
+from odoo.tests.common import SavepointCase
+
+
+class TestAccountInvoiceAutoSendByEmail(SavepointCase):
```

Comment:
Would be nice to use an accounting test case instead of having to spend to much code on creating accounts and journals in the setup.